### PR TITLE
Timestamp

### PR DESCRIPTION
**Pull request by frankvanrietimqs**
(Pull request is on top of master so you will see some of your own commits)

The idea here is to add a timestamp to assets and one to teh kv_type table. The timestamp gets updated every time an asset changes or a k/v table is changed. Note that for kv tables kv_type gets updated and not a timestamp on the specific kv_table. This is so a user can query one  table to see all lookup changes.

The timestamp is exposed in the asset dto (upon read). Not yet for k/v. It is also expose in all of the views. There are no tests yet for checking that the timestamp gets updated correctly. I would like you to do that.

### CODE REVIEW

File: Service/src/main/java/za/co/imqs/coreservice/controller/ExportController.java [hunk 1]
```diff
@@ -52,17 +56,25 @@ public ExportController(
...
     @RequestMapping(
             method = RequestMethod.GET, value = "/exporter/all-assets"
     )
-    public ResponseEntity exportCore() throws Exception {
+    public ResponseEntity exportCore(
+            @RequestParam(required = false, defaultValue ="false", name="exclude_last_changed") boolean exclLastChanged
+    ) throws Exception {
         final String extension = ".csv";
 
-        return zipExport(extension, ALL_ASSETS_QUERY);
```
Comment (CalebBredekamp) [inline Service/src/main/java/za/co/imqs/coreservice/controller/ExportController.java:65]: We can probably drop the ALL_ASSETS_QUERY now, unless you want to keep it for something else

Also, for what sort of thing would we not want to have the last_changed column?

File: Service/src/main/java/za/co/imqs/coreservice/controller/ExportController.java [hunk 2]
```diff
@@ -112,4 +124,18 @@ private ResponseEntity zipExport(String extension, String query) throws Exceptio
 
         return serveFile(Paths.get(filePath));
     }
+
```
Comment (CalebBredekamp) [inline Service/src/main/java/za/co/imqs/coreservice/controller/ExportController.java:127]: I'm a little confused as to whats happening here as well, What is this function doing exactly?

**Comment by CalebBredekamp** [comment]
This pr brought up a more general question for me as well:
What decided which view goes in which schema? Like there is an asset_core_view in public and and asset_core_internal in the asset schema which has more data. But how do you decide what goes where?

**Review by CalebBredekamp — APPROVED** [review]

File: Service/src/main/java/za/co/imqs/coreservice/controller/ExportController.java (hunk 1 above)
Comment (frankvanrietimqs) [reply Service/src/main/java/za/co/imqs/coreservice/controller/ExportController.java:65]: Agreed.  This is used by external systems. They have the ability to get all changes after the last time they queried the data.

**Pull request closed without merging**